test/node/connection-emit-actions.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+const test = require('tap');
+const net = require('net');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application');
+
+const application = new jstp.Application(app.name, app.interfaces);
+const serverConfig =
+  { applications: [application], authPolicy: app.authCallback };
```

Comment:
Sure. Btw we've got the same code in `connection-call.js`, We should probably also change it there.